examples/pytorch/speech-recognition/run_speech_recognition_seq2seq.py

```diff
@@ -336,7 +356,7 @@ def main():
             data_args.dataset_config_name,
             split=data_args.train_split_name,
             cache_dir=model_args.cache_dir,
-            use_auth_token=True if model_args.use_auth_token else None,
+            use_auth_token=True if model_args.token else None,
```

Comment:
If not set, i.e. `None`, the new behavior will be the same as in the current `main` (where we get argument set to `False`)

Reply:
We should pass along the token, and if tit's None, use the cached one if the user is logged in, like everywhere else.

Reply:
Oh yes, forget to change that. Thanks.

Reply:
@sgugger Let me know if the updated version works for you.

I just simply set `token=model_args.token,` and 

> use the cached one if the user is logged in

should be already done behind the scenes (right?)

Reply:
Yup!